### PR TITLE
Add dismiss instructions

### PR DESCRIPTION
**Pull request by 3lvis**
a

### CODE REVIEW

File: Demo/Helpers.swift [hunk 1]
```diff
@@ -36,3 +36,17 @@ extension IndexPath {
         }
     }
 }
+
+struct Helpers {
```
Comment (henrikholmsen) [inline Demo/Helpers.swift]: Maybe we could call this something like `Settings` instead of `Helpers`? 🤔

Comment (henrikholmsen) [reply Demo/Helpers.swift]: Then, perhaps, we could move the `lastSelectedIndexPath` in there as well.

Comment (3lvis) [reply Demo/Helpers.swift]: Settings sound something like preferences and this is just state. Maybe call it 	`State`?

Comment (henrikholmsen) [reply Demo/Helpers.swift]: True, `State` sounds right 😁

**Comment by 3lvis** [comment]
Changed one 🎆

**Pull request merged**